test/errors/incompatibleTypeAndInit-detailed.good

```diff
@@ -0,0 +1,9 @@
+─── error in incompatibleTypeAndInit.chpl:1 [IncompatibleTypeAndInit] ───
+  Type mismatch between declared type and initialization expression.
```

Comment:
I think this error should mention the name of the variable for which the mismatch occurs, perhaps like `Type mismatch between declared type of 'x' and initialization expression.` If that's too much information for the header, I think it should at least be in the message as `In the following declaration of 'x':`.

Reply:
This isn't always possible (this error is used for tuple declarations, too), but I'll make it happen in the case of variables.

Reply:
Why would it not be possible for a tuple declaration? I'm talking about writing the name of the variable, not its type, if that's the disconnect.

Reply:
Well, the whole tuple declaration is reported with the error; no one particular variable has the wrong type, the whole tuple does.

Reply:
Yep!